setup.py

```diff
@@ -51,7 +51,7 @@ def run_setup():
 
     setup(
         name='amazon.ion',
-        version='0.9.3',
+        version='0.10.3',
```

Comment:
Do we go to `0.10.1`?

Reply:
facepalm. yes.

Reply:
I even reviewed that and was like "yea looks right"